relationships/synthesis/AWS-LAMBDA-SPAN-to-DYNAMODB.yml

```diff
@@ -0,0 +1,17 @@
+relationships:
+  - name: lambdaCallsDynamoDb
+    version: "1"
+    origins:
+      - AWS Integration
+    conditions:
+      - attribute: name
+        regex: "*dynamodb*"
```

Comment:
is this the best condition we can get? how can you identify from the telemetry that it comes from dynamo other than this regex?